Stratis.Bitcoin/Stratis.Bitcoin.csproj

```diff
@@ -18,7 +18,11 @@
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
There is a learning curve with git but it becomes really awesome when you know how to use it